db/migrate/20240813141845_rename_centos_stream_mirror_to_centos.rb

```diff
@@ -0,0 +1,9 @@
+class RenameCentosStreamMirrorToCentos < ActiveRecord::Migration[6.1]
+  def up
+    Medium.unscoped.where(name: "CentOS Stream 9 mirror")&.update_all(name: "CentOS mirror")
+  end
+
+  def down
+    Medium.unscoped.where(name: "CentOS mirror")&.update_all(name: "CentOS Stream 9 mirror")
```

Comment:
I don't think `.where` can return `nil` so there should be no need for the safe operator
```suggestion
    Medium.unscoped.where(name: "CentOS Stream 9 mirror").update_all(name: "CentOS mirror")
  end

  def down
    Medium.unscoped.where(name: "CentOS mirror").update_all(name: "CentOS Stream 9 mirror")
```